cortex/model.py

```diff
@@ -15,9 +15,9 @@
 """
```

Comment:
Need to figure out how to explain this versus experiments ...

Reply:
Added some more context into the docstring.